src/Microsoft.DotNet.Build.Tasks/ExecWithRetriesForNuGetPush.cs

```diff
@@ -17,9 +20,18 @@ namespace Microsoft.DotNet.Build.Tasks
     /// </summary>
     public class ExecWithRetriesForNuGetPush : BuildTask
     {
+        private const int DownloadComparisonBufferBytes = 1024 * 1024;
+
         [Required]
         public string Command { get; set; }
 
+        /// <summary>
+        /// Package file that is pushed by the given command. Only used if an
+        /// IgnoredErrorMessagesWithConditional item is specified with ConditionalIfV2FeedIdentical
```

Comment:
This "feels" like a complicated way to enable things.  Perhaps it's just because I don't understand where this is being used.

Reply:
You might be right--it felt to me like a natural way to extend the existing functionality and avoid hard-coding the 409 error message, but I'll revisit this.

Reply:
The only thing I could think to do is remove the new metadata and add two new properties instead: the v2 feed to look at and the error message that triggers checking that feed. Barely different at all--maybe I'm mentally locked into thinking about this task this way. Any suggestions?

Reply:
The presence of a single `PassIfIdenticalV2Feed` property seems like it would be sufficient.  ProdCon would just set this value.  That might be simpler than adding metadata to every item.  I think both properties is overkill.  Unless there are conditions where some packages want this condition and others do not.

Reply:
With only that property and no "the error message that triggers checking that feed" property, we would have to hard-code the error message, right? I've been thinking we want to avoid that so BuildTools users can react more rapidly than taking a BuildTools fix if the error message changes for some reason.

Reply:
I see.  I was thinking it would just fail, then check for identicalness, then pass if identical (regardless of what the error message was).  If the specific error message check is necessary, then that does complicate it.  Do you care about the error message if they're identical?

Reply:
Hmm, the idea was to avoid taking the time to download when there's a different kind of error that seems unrelated. But it's true that if the feed has the package already, the error doesn't matter no matter what kind it is. I agree, I'll make this change.